bl-user-setup

```diff
@@ -18,15 +18,15 @@
 #    along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 # USER="$1" USER is exported by lightdm
```

Comment:
I had a look at the lightdm source and it looks indeed like it sets HOME based on the passed user name (file: src/seat.c, L371) so we can use that. This still means that mentionings of `/home/$USER` must be changed to `$HOME`.

BTW the variables set in the script are 
```c
    if (user)
    {
        process_set_env (script, "USER", user_get_name (user));
        process_set_env (script, "LOGNAME", user_get_name (user));
        process_set_env (script, "HOME", user_get_home_directory (user));
    }
```
plus `$PATH`.